### PR TITLE
APP-162: [C] Basic level Apps: Endpoint ownership

### PR DESCRIPTION
**Pull request by Dubland**
/cc @Slash32 @RostakaGmfun @vadimol

### CODE REVIEW

File: verifiersdemo/source/c/src/kaa_demo.c [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+
+#include <kaa_error.h>
+#include <platform/kaa_client.h>
+#include <kaa_configuration_manager.h>
+#include <kaa_user.h>
+#include <kaa_profile.h>
+#include <gen/kaa_verifiers_demo_event_class_family.h>
+#include <platform/ext_key_utils.h>
+
+extern kaa_error_t kaa_profile_manager_set_endpoint_access_token(kaa_profile_manager_t *self, const char *token);
```
Comment (vadimol) [inline verifiersdemo/source/c/src/kaa_demo.c]: why _extern_? You have already included _kaa_profile.h_

Comment (rasendubi) [reply verifiersdemo/source/c/src/kaa_demo.c]: No external function declarations outside of `.h` files, please

Comment (Dubland) [reply verifiersdemo/source/c/src/kaa_demo.c]: Done

**Review by vadimol — APPROVED** [review]
LGTM

**Review by Slash32 — APPROVED** [review]

**Pull request merged**